default.yml

```diff
@@ -40,23 +35,17 @@ Layout/HashAlignment:
 Layout/MultilineOperationIndentation:
   Enabled: false
 
+# https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/Debugger
+# Show linting errors on debugger/binding.pry even in development
+Lint/Debugger:
+  Enabled: true
```

Comment:
[Docs](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/Debugger)

I think it is useful to have rubocop complain about this, even if CI will fail. The time I have these in my code _should_ be short lived, and stick out so I remember to remove them.